spec/models/invoice_spec.rb

```diff
@@ -13,10 +13,14 @@
   let!(:customer_1) {Customer.create!(first_name: "Billy", last_name: "Joel")}
 
   let!(:invoice_1) {customer_1.invoices.create!(status: 1, created_at: '2012-03-25 09:54:09')}
+  let!(:invoice_2) {customer_1.invoices.create!(status: 1, created_at: '2012-03-25 09:54:09')}
+  let!(:invoice_3) {customer_1.invoices.create!(status: 1, created_at: '2012-03-25 09:54:09')}
 
   let!(:invoice_item_1) {InvoiceItem.create!(quantity: 3, unit_price: item_1.unit_price, item_id: item_1.id, invoice_id: invoice_1.id, status: 0)}
   let!(:invoice_item_2) {InvoiceItem.create!(quantity: 5, unit_price: item_2.unit_price, item_id: item_2.id, invoice_id: invoice_1.id, status: 0)}
   let!(:invoice_item_3) {InvoiceItem.create!(quantity: 1, unit_price: item_3.unit_price, item_id: item_3.id, invoice_id: invoice_1.id, status: 0)}
+  let!(:invoice_item_4) {InvoiceItem.create!(quantity: 1, unit_price: item_3.unit_price, item_id: item_3.id, invoice_id: invoice_2.id, status: 1)}
+  let!(:invoice_item_5) {InvoiceItem.create!(quantity: 1, unit_price: item_3.unit_price, item_id: item_3.id, invoice_id: invoice_3.id, status: 2)}
 
```

Comment:
Are we testing for the status of invoices or invoice_items? When creating an invoice_item and assigning a status, we are assigning: `["packaged", "pending", "shipped"]`, and with all three invoices completed -- we're not testing for the cancelled or in progress right? I could be reading the user story wrong.

Reply:
All three of the items that are 'packaged' are associated with invoice_1, the item that is 'pending' is associated with invoice_2, and the item that is 'shipped' is associated with invoice_3.

I set it up this way so that once invoice has multiple instances of items that matched the criteria (invoice_1), and that all three statuses were associated with different invoices so that 'incomplete_invoices' would return a single instance of invoice_1, invoice_2, and exclude invoice_3.

Reply:
Ahh, gotcha. Thank you for the clarification!